sceptre/resolvers/project_variables.py

```diff
@@ -2,6 +2,9 @@
 
 import os
 import yaml
+from warnings import warn, simplefilter
+
```

Comment:
We tend to put a newline in between stdlib imports and external imports [e.g.](https://github.com/cloudreach/sceptre/blob/master/sceptre/cli.py#L17)